build.rb

```diff
@@ -17,14 +17,18 @@ def pipe(command)
   output
 end
 
-def fetch(url)
+def fetch(url, name = nil)
```

Comment:
Would be better as a named argument, also maybe `patch_name` instead of `name`.

Reply:
Well, `name` is used here b/c in theory you could name other stuff since it's not just specific to patches.

Reply:
In theory, using keyword argument means we can't built cedar binaries, which is probably ok.

Reply:
ya, cedar is read only mode.

Instead of name, maybe `file_name`.